LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/client/LabClient.java

```diff
@@ -483,11 +481,10 @@ public boolean enablePolicy(@NonNull final String upn, @NonNull final Protection
     public boolean disablePolicy(@NonNull final String upn, @NonNull final ProtectionPolicy policy) {
         final DisablePolicyApi disablePolicyApi = new DisablePolicyApi();
         try {
-            final CustomSuccessResponse customSuccessResponse = disablePolicyApi.apiDisablePolicyPut(upn, policy.toString());
+            final String disablePolicyResponse = disablePolicyApi.apiDisablePolicyPost(upn, policy.toString());
```

Comment:
Looks like these are changing from PUT to POST

Reply:
Yes, the new api for these calls uses Post

Reply:
Temp user creation already used POST to begin with, i'm not totally understanding why reset password, enable policy etc were using PUT with the old api